payments/enums.py

```diff
@@ -125,3 +125,8 @@ class PricingCategory(IntegerChoices):
     DINGHY = 1, _("Dinghy places")
     TRAILER = 2, _("Trailer places")
     VASIKKASAARI = 3, _("Vasikkasaari berths")
+
+
+class TalpaProductType(TextChoices):
+    BERTH = "west", _("West")
+    WINTER = "east", _("East")
```

Comment:
This is probably a typo 😅 but you have `BERTH = "west"` and `WINTER = "east"`

Reply:
yes, fixed!